code/datums/components/crafting/recipes.dm

```diff
@@ -48,9 +48,40 @@
 /datum/crafting_recipe/stunprod
 	name = "Stunprod"
 	result = /obj/item/weapon/melee/baton/cattleprod
-	reqs = list(/obj/item/weapon/handcuffs/cable = 1,
-				/obj/item/stack/rods = 1,
-				/obj/item/weapon/tool/wirecutters = 1)
+	reqs = list(list(/obj/item/weapon/handcuffs/cable = 1),
```

Comment:
Nit: This is uncomfortable. Not sure how else you'd do alternatives without switching this to some kind of datum system but nested lists are a bit urgh generally.